discovery-provider/chain/health.py

```diff
@@ -0,0 +1,24 @@
+import requests
+
+health_url = "http://localhost:8545/health"
+rpc_url = "http://localhost:8545"
+
+health_res = requests.get(health_url)
+health_json = health_res.json()
+
+healthy = True
+
+health_check_node = health_json.entries["node-health"]
+health_check_description = health_check_node.description
+health_check_healthy = health_json.status == "Unhealthy"
+
+if health_check_healthy:
+    healthy = False
+
+not_producing_msg = "The node stopped producing blocks"
+if health_check_healthy and not_producing_msg in health_check_description:
+    healthy = True
```

Comment:
won't this [change](https://github.com/AudiusProject/audius-docker-compose/pull/219) result in both signers/non-signers reporting healthy?